### PR TITLE
Describe the method to add stored payment method from checkout

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) describes the method to add stored payment method from checkout

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/customers/account-dashboard-stored-payment-methods.html

### CODE REVIEW

File: src/customers/account-dashboard-stored-payment-methods.md [hunk 1]
```diff
@@ -8,3 +8,23 @@ A payment method that supports a secure vault, such as [Braintree]({% link payme
 
 ![]({% link images/images-ee/customer-account-stored-payment-methods.png %}){: .zoom}
 _Stored Payment Methods_
+
+## Add stored payment method from checkout
```
Comment (leslietilling) [inline src/customers/account-dashboard-stored-payment-methods.md:12]: The Admin user guide is written for merchants, so describing storefront procedures are customer third-person.

File: src/customers/account-dashboard-stored-payment-methods.md [hunk 2]
```diff
@@ -8,3 +8,23 @@ A payment method that supports a secure vault, such as [Braintree]({% link payme
...
+
+1. From the storefront, go to the detail page of the product.
```
Comment (leslietilling) [inline src/customers/account-dashboard-stored-payment-methods.md:14]:
```suggestion
1. From the storefront, the customer goes to the detail page of the product.
```

File: src/customers/account-dashboard-stored-payment-methods.md [hunk 3]
```diff
@@ -8,3 +8,23 @@ A payment method that supports a secure vault, such as [Braintree]({% link payme
...
+
+1. Add product to the cart.
```
Comment (leslietilling) [inline src/customers/account-dashboard-stored-payment-methods.md:16]:
```suggestion
1. Adds product to the cart.
```

File: src/customers/account-dashboard-stored-payment-methods.md [hunk 4]
```diff
@@ -8,3 +8,23 @@ A payment method that supports a secure vault, such as [Braintree]({% link payme
...
+
+1. Proceeds to checkout.
+
+1. Complete the **Shipping** step.
```
Comment (leslietilling) [inline src/customers/account-dashboard-stored-payment-methods.md:20]:
```suggestion
1. Completes the **Shipping** step.
```

File: src/customers/account-dashboard-stored-payment-methods.md [hunk 5]
```diff
@@ -8,3 +8,23 @@ A payment method that supports a secure vault, such as [Braintree]({% link payme
...
+
+1. Select the **Braintree Credit Card** payment method.
```
Comment (leslietilling) [inline src/customers/account-dashboard-stored-payment-methods.md:22]:
```suggestion
1. Selects the **Braintree Credit Card** payment method.
```

File: src/customers/account-dashboard-stored-payment-methods.md [hunk 6]
```diff
@@ -8,3 +8,23 @@ A payment method that supports a secure vault, such as [Braintree]({% link payme
...
+
+1. Fill in credit card data.
```
Comment (leslietilling) [inline src/customers/account-dashboard-stored-payment-methods.md:24]:
```suggestion
1. Fills in credit card data.
```

File: src/customers/account-dashboard-stored-payment-methods.md [hunk 7]
```diff
@@ -8,3 +8,23 @@ A payment method that supports a secure vault, such as [Braintree]({% link payme
...
+
+1. Choose **Save for later use** check-box.
```
Comment (leslietilling) [inline src/customers/account-dashboard-stored-payment-methods.md:26]:
```suggestion
1. Selects the **Save for later use** checkbox.
```

File: src/customers/account-dashboard-stored-payment-methods.md [hunk 8]
```diff
@@ -8,3 +8,23 @@ A payment method that supports a secure vault, such as [Braintree]({% link payme
...
+
+1. Click <span class="btn">Place Order</span>.
```
Comment (leslietilling) [inline src/customers/account-dashboard-stored-payment-methods.md:28]:
```suggestion
1. Clicks <span class="btn">Place Order</span>.
```

File: src/customers/account-dashboard-stored-payment-methods.md [hunk 9]
```diff
@@ -8,3 +8,23 @@ A payment method that supports a secure vault, such as [Braintree]({% link payme
...
+
+The saved payment method will be displayed in the **Stored Payment Methods** tab of the customer dashboard.
```
Comment (leslietilling) [inline src/customers/account-dashboard-stored-payment-methods.md:30]:
```suggestion
The saved payment method is then displayed in the _Stored Payment Methods_ tab of the customer dashboard.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling , I have implemented the suggestions you describe, could you please approve? 

Thank you!

File: src/customers/account-dashboard-stored-payment-methods.md (hunk 9 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-stored-payment-methods.md]: Done ✅

File: src/customers/account-dashboard-stored-payment-methods.md (hunk 8 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-stored-payment-methods.md]: Done ✅

File: src/customers/account-dashboard-stored-payment-methods.md (hunk 7 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-stored-payment-methods.md]: Done ✅

File: src/customers/account-dashboard-stored-payment-methods.md (hunk 6 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-stored-payment-methods.md]: Done ✅

File: src/customers/account-dashboard-stored-payment-methods.md (hunk 5 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-stored-payment-methods.md]: Done ✅

File: src/customers/account-dashboard-stored-payment-methods.md (hunk 4 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-stored-payment-methods.md]: Done ✅

File: src/customers/account-dashboard-stored-payment-methods.md (hunk 3 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-stored-payment-methods.md]: Done ✅

File: src/customers/account-dashboard-stored-payment-methods.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/account-dashboard-stored-payment-methods.md]: Done ✅

**Comment by magento-engcom-team** [comment]
@leslietilling, an error occurred during the Pull Request import.

**Comment by magento-engcom-team** [comment]
@leslietilling the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-merchdocs-1219`.

**Comment by leslietilling** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1219`.

**Pull request merged**